java/packagescanner_test.go

```diff
@@ -10,6 +15,35 @@ import (
 	"github.com/quay/claircore/test"
 )
 
+// TestDisableAPI runs the java scanner over some layers known to have java
+// packages installed.
+func TestDisableAPI(t *testing.T) {
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		t.Error("http request called")
```

Comment:
nit: maybe be kind to a future operator by explaining why this is an error

Reply:
Also, this should be `t.Fatal()` if the API is ever called, we don't care about results parity after this error.

Reply:
re: explaining the error, lmk if the new changes are still unclear. Definitely want to value clearer messages in the tests